CETCompression.md

```diff
@@ -368,6 +370,43 @@ def groupByIgnoringDigits(start: Long, end: Long, base: Int, numDigits: Int): Ve
 }
 ```
 
+## Adaptor Point Computation Optimizations
+
+Computing a large number of adaptor points for events with outcome composed of many digits is a computationally intensive process.
+To reduce the time and resources required, some optimizations can be applied.
+
+### Pre-computing
+
+The first optimization is to first compute the signature points for the possible value of each digit.
+For example, for an event with outcomes decomposed in base 2 with 10 digits, first compute:
+```
+S0_0 = R0 + 0 * P
+S0_1 = R0 + 1 * P
+S1_0 = R1 + 0 * P
+S1_1 = R1 + 1 * P
+...
+S9_0 = R9 + 0 * P
+S9_1 = R9 + 1 * P
+```
+
+Then compute the aggregate signature points by combining the Si_j.
+For example, to compute the signature point for the value `0 0 0 0 0 0 0 0 0 0`, compute:
+```
+S = S0_0 + S1_0 + ... + S9_0
+```
```

Comment:
probably worth a second example just to drive home how this is actually an optimization and not just a round-about way of computing the same thing

Reply:
Added a second example